kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -76,7 +76,7 @@ domain.weblogic.oracle/domain1 patched
 [INFO] Successfully patched domain 'domain1' in namespace 'weblogic-domain-1' with 'NEVER' start policy!
 ```
 
-### Scripts to scale a WebLogic cluster
+### Script to scale a WebLogic cluster
 
```

Comment:
The title of this doc should be: Domain life cycle sample scripts. (life cycle (n), lifecycle (adj))
after it's value -> after its value